applications/GeoMechanicsApplication/tests/cpp_tests/custom_constitutive/test_incremental_linear_elastic_interface_law.cpp

```diff
@@ -158,6 +197,18 @@ KRATOS_TEST_CASE_IN_SUITE(WhenAnInitialStateIsGivenStartFromThereAfterMaterialIn
     KRATOS_EXPECT_VECTOR_RELATIVE_NEAR(value, initial_traction, relative_tolerance)
 }
 
+KRATOS_TEST_CASE_IN_SUITE(TryingToGetTheValueOfAnUnsupportedVectorVariableRaisesAnError,
```

Comment:
It might make sense to have the two test for 'throws` together, since they are so similar. On the other hand, one is for GetValue, the other for CalculateValue, so I'm fine with grouping it like this as well

Reply:
My reasoning was to have one test case for each failing member function. If at some point we would add overloads for these members, we can add tests for them in the corresponding test cases. Do you have a preference for either this approach or your suggestion? If yes, why? Thanks.

Reply:
As discussed, it's more about the grouping about the test cases (not merging them), but since it is just a matter of preference (and I don't have a strong opinion about it either), let's just leave it as is